tests/onnxruntime/test_modeling.py

```diff
@@ -3491,6 +3522,8 @@ def test_compare_to_io_binding(self, test_name: str, model_arch: str, use_cache:
     def test_compare_generation_to_io_binding(
         self, test_name: str, model_arch: str, use_cache: bool, use_merged: bool
     ):
+        if model_arch == "encoder-decoder":
+            use_cache = False
```

Comment:
There should rather be a skipTest for the use_cache case.

Reply:
In this particular test,`use_cache=True` ids only tested so the model was never going to be tested. Once, for the particular model I changed it to `False`
```python
    @parameterized.expand(
        grid_parameters({"model_arch": SUPPORTED_ARCHITECTURES, "use_cache": [True], "use_merged": [False, True]})
    )
```